pandas/core/arrays/datetimelike.py

```diff
@@ -118,7 +118,7 @@
 from pandas.core.algorithms import (
     checked_add_with_arr,
     isin,
-    mode,
+    mode as mode_func,
     unique1d,
 )
```

Comment:
in a few other files, there is
```python
import pandas.core.algorithms as algos
```

shall we just do that and then call `algos.mode`, etc...?

Reply:
Sure, Thanks!

Reply:
Hey are these two algos the same ?
```python
from pandas._libs import (
    algos,
    lib,
)
```
```python
from pandas.core.algorithms import algos

```

Reply:
ah good catch - let's just do
```
from pandas.core import algorithms 
```
and then `algorithms.mode`, etc...

Reply:
Done